pkg/githubutils/types.go

```diff
@@ -0,0 +1,44 @@
+package githubutils
+
+type Author struct {
```

Comment:
yeah, you shouldn't need to define these if you use a library.

Reply:
Indeed. But I think for now it is an acceptable tradeoff considering the simple use case.